testsuite/features/step_definitions/navigation_steps.rb

```diff
@@ -547,7 +547,10 @@
 
 Then(/^I should be logged in$/) do
   xpath_query = "//a[@href='/rhn/Logout.do']"
-  raise 'User is not logged in' unless find(:xpath, xpath_query)
+  unless find(:xpath, xpath_query)
```

Comment:
But.. that would behaves the same than using timeout parameter with the double of the default capybara timeout. 🤔